source/includes/gho/_api.md

```diff
@@ -0,0 +1,302 @@
+department | [Department](#department) | 
+documents | [Document](#document) |     These are documents that either came over from Greenhouse Recruiting, were attached directly to the employee    profile, or attached to a task.  This does _not_ include E-Signature requests.  
+email | [String](#string) | The employee's work email.  They need this in order to sign in
+employment_status | [EmploymentStatus](#employmentstatus) | 
+first_name | [String](#string) | 
+hr_manager | [Employee](#employee) | The employee's HR Manager.
+id | [ID](#id) | 
+last_name | [String](#string) | 
+location | [Location](#location) | 
+manager | [Employee](#employee) | This employee's direct manager.
+middle_name | [String](#string) | 
+personal_email | [String](#string) | The employee's personal email.
+phone_number | [String](#string) | 
+preferred_first_name | [String](#string) |     This is the name that your employee prefers to go by.  If this value is set, Greenhouse Onboarding will display    this name everywhere in the product instead of the employee's legal name.  
+preferred_last_name | [String](#string) | 
+profile_image | [File](#file) |     A file containing the employee's profile image.  This image is displayed in emails, reports and directory pages.  
+signature_requests | [SignatureRequest](#signaturerequest) |     These are E-Signature requests initiated through Greenhouse Onboardinging.  Keep in mind that these requests can    be in a number of different states in their lifecycle and may not always have a signed document available to    download.  
+start_date | [Date](#date) | 
+suffix | [String](#string) | 
+title | [String](#string) | The employee's job title.
+updated_at | [DateTime](#datetime) | 
+work_country_code | [String](#string) | 
+
+## EmployeeConnection
+The connection type for Employee.
+
+Field | Type | Description
+--------- | ----------- | -----------
+edges | [EmployeeEdge](#employeeedge) | A list of edges.
+pageInfo | [PageInfo](#pageinfo) | Information to aid in pagination.
+
+## EmployeeEdge
+An edge in a connection.
+
+Field | Type | Description
+--------- | ----------- | -----------
+cursor | [String](#string) | A cursor for use in pagination.
+node | [Employee](#employee) | The item at the end of the edge.
+
+## File
+A File record
+
+Field | Type | Description
+--------- | ----------- | -----------
+expires_at | [DateTime](#datetime) | The time when the URL will expire.  After this time, you will need to generate a new URL.
+file_name | [String](#string) | The original name of the file.
+file_size | [Int](#int) | The file size, in bytes
+file_url | [String](#string) | An expiring URL you can use to download the file.
+
+## Location
+    Represents a single location in your company.  Employees can belong only belong to a single Location.  It is    possible for employees to not belong to any Location.  Lepartments are used in a variety of ways in Greenhouse    Onboarding, including permissions and onboarding plans  
```

Comment:
Same comment as line 30.

"Employees may belong to zero or one location."

"Lepartments"

Are locations actually used to for permissions or was this copypasta from department?

Reply:
😂  

They actually are used in permissions.